File: dastanlib/src/main/java/com/dastanapps/dastanlib/utils/ResourceUtils.java
package com.dastanapps.dastanlib.utils;

import android.content.Context;

/**
 * Created by Admin on 17-Aug-17.
 */

public class ResourceUtils {

    public static int getDrawableResId(Context context, String filename) {
        return context.getResources().getIdentifier(filename,
                "drawable", context.getPackageName());
    }
}

File: common/src/main/java/com/dastanapps/dastanlib/utils/ResourceUtils.kt
package com.dastanapps.dastanlib.utils

import android.content.Context

/**
 * Created by Iqbal Ahmed on 17-Aug-17.
 */

object ResourceUtils {

    fun getDrawableResId(context: Context, filename: String): Int {
        return context.resources.getIdentifier(filename,
                "drawable", context.packageName)
    }
}
